installer/windows/invest_installer.nsi

```diff
@@ -298,7 +298,7 @@ FunctionEnd
 
 Section "InVEST Tools" Section_InVEST_Tools
     AddSize 230793  ; This size is based on Windows build of InVEST 3.4.0
-    SetShellVarContext all
+    SetShellVarContext current 

```

Comment:
Mainly out of curiosity, did this change anything about which parts of the filesystem are used? Either the install location or the application's data (e.g. `AppData/Local/Roaming/...` or whatever the heck that path is)?

Reply:
Oh that's a great question Dave! I don't think this should effect that. But `SetShellVarContext` set to `all` meant that the user could not write the startup menu shortcuts in `ProgramData` because they didn't have permission to do this for All Users. 

We should definitely test what happens when running InVEST models and where we normally expect these folders to be.

Do we have a good handle on all the local paths InVEST uses for temp folders, caches, etc... ?

Reply:
Oh, here's the resource: https://nsis.sourceforge.io/Reference/SetShellVarContext

Reply:
@davemfish I ran some models and checked cache folders. Folder being used:
>../AppData/Roaming/Natural Capital Project/

This seems to line up with what I'm seeing on my admin side.

For some reason I thought the non-admin user was also having start up shortcuts go to `ProgramData`, but apparently not. Which makes sense since they don't have permission to write there. Instead they are going to:
>.../AppData/Roaming/Microsoft/Windows/Start Menu/Programs/

So, I think everything is okay.

Reply:
Thanks for checking on that! I think everything is okay, too.

Reply:
Oooo I think we might have a pickle here: institutional users.  Perrine wrote to me a few months back about installing InVEST in a computer lab for a class.  IT was going to perform the install.  If the Start Menu shortcuts will be written to `C:\Users\<IT User>\AppData\...`, then anyone _other_ than IT will need to launch the application from the CLI.

Some installers out there have an option to "Install for all users" and then "install for this user only" ... maybe NSIS has a standard way to do something similar?